index.html

```diff
@@ -0,0 +1,347 @@
+    <link rel = "stylesheet" href = "style.css">
+  </head>
+
+  <body>
+    <header>
+      <nav class = "navigation">
+
+        <div class = "menuLink">
+          <div class = "logo">
+            <a href = "#"><img src = "images/nav-burger.png" alt = "menu"></a>
+          </div>
+          <div class = "logo">
+            <a href = "#"><img src = "images/nav-logo.png" alt = "YTlogo"></a>
+          </div>
+        </div>
+
+        <div class = "search">
+          <input class = "search_text" type = "text" name = "search" placeholder = "Search">
+          <button class = "search_icon" type = "button" name = "button">
+            <img class= "search_img" src = "images/nav-search.png" alt = "SearchIcon">
+          </button>
+        </div>
+
+        <div class = "headerRightIcons">
+          <a href = "#"><img src = "images/nav-stream.png" alt = "stream"></a>
+          <a href = "#"><img src = "images/nav-apps.png" alt = "apps"></a>
+          <a href = "#"><img src = "images/nav-share.png" alt = "share"></a>
+          <a href = "#"><img src = "images/nav-notification.png" alt = "notifications"></a>
+          <a href = "#"><img src = "images/nav-profile.jpg" alt = "profilePic"></a>
+        </div>
+      </nav>
+    </header>
+
+    <div class="content">
+      <section>
+          <div class = "iframe-container">
+            <iframe src = "https://www.youtube.com/embed/5MgBikgcWnY"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen>
+           </iframe>
+          </div>
+          <div class= "videoTitle">
+              <h4>The first 20 hours -- how to learn anything | Josh Kaufman | TEDxCSU</h4>
+          </div>
+          <div class="videoIcons">
+            <div class="video_views">
+              <p>15.289.579 Views</p>
+            </div>
+            <div >
+              <a href="#"><img src="images/footing-socialicons.png" alt="socialIcons"></a>
+            </div>
+          </div>
+          <hr/>
+            <div class="authorSection">
+              <div class="uploaderPic">
+                <a href="#"><img src="images/comments-profile.jpg" alt="uploader pic"></a>
+              </div>
+
+              <div class="author">
+                <h5>TEDx Talks</h5>
+                <h3>Published on Feb 27, 2014<br><br><br><br></h3>
+                  <p>Never miss a talk! SUBSCRIBE to the TEDx channel:
+                    <a href="http://bit.ly/1FAg8hB"> http://bit.ly/1FAg8hB</a><br><br>Josh Kaufman
+                    is the author of the #1 international bestseller, 'The Personal MBA: Master the Art of Business',
+                    as well as the upcoming book 'The First 20 Hours: Mastering the Toughest Part of
+                    Learning Anything.' </p>
+                  <h2><br><br>VIEW MORE</h2>
+              </div>
+              <div class="subscribe">
+                <a href="#"><img src="images/subscribe.png" alt="subsImg"></a>
```

Comment:
use the 1<button> tag here instead of the `<a>` tag :+1:

Reply:
Thank you very much !! 
We have doubts about the tag "1" because we don't find it.

Reply:
Do you mean we should replace the <a> tag with any other tag?

Reply:
Sinppet was not showing I edited it.

Reply:
thank you! it is done ^^

Reply:
Ok, the adjustments are done could you review them please?